docs/README.Windows.md

```diff
@@ -220,6 +220,11 @@ Or configure build for 32bit:
 cmake -G "Visual Studio 17 2022" -A Win32 -T host=x64 %userprofile%\kodi
 ```
 
+Or configure build for ARM 64bit:
+```
+cmake -G "Visual Studio 17 2022" -A arm64 -T host=x64 %userprofile%\kodi
```

Comment:
I build WIN-ARM64 on WIndows arm64 (parallels vm on Macbook M2). We should probably remove all `-T` flags